serverless.yml

```diff
@@ -0,0 +1,125 @@
+service: JPC-Messaging-Service
+
+plugins:
+  - serverless-python-requirements
+
+custom:
+  serviceName: JPC-Messaging-Service
```

Comment:
You already have the service name. Its on line 1

Reply:
I wasn't sure if I should name the service in the custom field or not, so just went for it. After looking at docs again,  The LOG_LEVEL by default is INFO so it is probably redundant to specify that as well in the Environment.Variables.

Reply:
This still exists and is not necessary.